djangocon_2020/content/conduct/code_of_conduct/code_of_conduct.md

```diff
@@ -102,8 +101,6 @@ In your report please include, when possible:
 
 If you don't have some of this information, or not at this time, please still make the report anyways. You can contact us at any time after your report if you want to add, edit or take back any information you shared.
 
-If you feel unsafe reporting in person, you may choose someone to represent you. In this case, we'd need their contact information, but we'd ask this person to make clear that they are not reporting in their own name.
```

Comment:
Why is this not relevant anymore ?

Reply:
after some discussion, we decided to keep that option for someone who wants to make a report so we will add that text again to our CoC